pkg/downloader/chart_downloader.go

```diff
@@ -184,11 +183,11 @@ func (c *ChartDownloader) getOciURI(ref, version string, u *url.URL) (*url.URL,
 //
 // A version is a SemVer string (1.2.3-beta.1+f334a6789).
 //
-//	- For fully qualified URLs, the version will be ignored (since URLs aren't versioned)
-//	- For a chart reference
-//		* If version is non-empty, this will return the URL for that version
-//		* If version is empty, this will return the URL for the latest version
-//		* If no version can be found, an error is returned
+//   - For fully qualified URLs, the version will be ignored (since URLs aren't versioned)
+//   - For a chart reference
+//   - If version is non-empty, this will return the URL for that version
+//   - If version is empty, this will return the URL for the latest version
+//   - If no version can be found, an error is returned
```

Comment:
I think vscode reformatted this

Reply:
Yeah it annoyingly does it on save. Have it back as the original now

Reply:
This is probably due to go 1.20 using markdown for comments. A resave of the files probably triggers a go fmt which then generates this.

Reply:
I have the same problem with this file and vscode. :smirk: